src/tests/components/experiments/ExperimentTable.test.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Adapter from "enzyme-adapter-react-16";
+import { shallow, configure } from "enzyme";
+configure({ adapter: new Adapter() });
+import { fromJS } from "immutable";
+import { ExperimentTable } from "../../../components/experiments/ExperimentTable";
+
+describe("ExperimentTable component", () => {
+    const container = shallow(
+        <ExperimentTable 
+          filteredExperiments={fromJS([{name: "Name", start_date: "2018-08-22", end_date: "2018-08-23", status: "live"}])}
+        />
+      );
+    
+    it("should render an experiment table with a correct header row defining the head of the columns of the table", () => {
```

Comment:
This is better but it's still only one setup and then a bunch of assertions about it, so isn't it still basically just one test?